tests/unit/requirements.txt

```diff
@@ -1 +1,2 @@
 pytest
+freezegun
```

Comment:
Is this really required? Could we use mock instead?

Reply:
I am okay with this. It makes the unit test easy to understand.

Reply:
Mocking `datetime.datetime.now()` directly is not possible because it's implemented in C. We could have a workaround, like creating a wrapper object around the datetime object and mocking it. But I think using this tool is simpler.